stock_picker.rb

```diff
@@ -0,0 +1,22 @@
+
+def stock_picker(prices_array)
+    highest_interval = 0
+    buy_day = 0
+    sell_day = -1
```

Comment:
naming
what are you doing?
you are saving 3 variables outside the loop with:
* the highest profit you've found
* the buying day for this profit
* the selling day for this profit

so 
* highest interval - it's not an interval it's the profit
* buy/sell_day - it's not just any buying or seeling day

```suggestion
    highest_profit = 0
    highest_profit_buy_day = 0
    highest_profit_sell_day = -1
```

another option - that's even better
who said you should save the sell and buy day separately?
save them into one array
```suggestion
    highest_profit = 0
    highest_profit_interval = [0, 0]
```

Reply:
done